pkg/apis/core/types.go

```diff
@@ -988,6 +988,30 @@ type CinderVolumeSource struct {
 	// Ex. "ext4", "xfs", "ntfs". Implicitly inferred to be "ext4" if unspecified.
 	// +optional
 	FSType string
+	// Optional: SecretRef is reference to the connection information for openstack, default is empty.
+	// +optional
+	SecretRef *LocalObjectReference
```

Comment:
the comment is different from that in v1/types.go

Reply:
Fixing